_posts/2016-08-18-emerging-docker.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: Running Docker Natively on Windows and Mac
+---
+
+[Docker](https://www.docker.com) is an emerging technology for packaging software in so-called containers that can run processes isolated from the rest of the operating system. It is the midway point between a complete virtual machine for a single system and running a process with limited user rights.
+
+Docker provides so-called software images that package software. These images can then be executed inside a docker container. Furthermore, Docker provides a service called [DockerHub](https://hub.docker.com/) that allows users to upload and publish their own images. This allows developers to bundle their software and distribute it easily. Furthermore, since each container runs isolated from the rest of the systems, the developers do not have to rely on any kind of other system configuration.
+
+Within the ODK project this allows us to bundle mathematical software systems (such as [GAP](http://www.gap-system.org/), [Sage](http://www.sagemath.org/), ...) and distribute them in a manner that is both accessible to users and easily maintainable for developers. Furthermore the isolated aspect of docker containers allows us to easily integrate multiple systems together without having to make additional assumptions about the users specific setup -- we can just run all systems in one docker container. This way users can install the entire mathematical software stack that the OpenDreamKit project aims to provide easily.
```

Comment:
Maybe replace `, ...` by "and others" or "etc." - otherwise looks too much like a draft